.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,6 +19,7 @@ https://github.com/zfsonlinux/zfs/wiki/Buildbot-Options
 <!--- Include details of your testing environment, and the tests you ran to -->
 <!--- see how your change affects other areas of the code, etc. -->
 <!--- If your change is a performance enhancement, please provide benchmarks here. -->
+<!--- If no zfstest and/or zloop has been run, please submit it as a draft-PR -->
```

Comment:
As mentioned above I don't think we want to make this a stipulation, let's drop this addition.

Reply:
Whoops, indeed this should've been removed in the previous itterations when I removed it being required. Thanks for noticing.

Reply:
I've removed the draft PR reminder from the checklist and replaced this with said softer reminder.
I think that does suffice.